### DISCUSSION TITLE
Enhancement: Sync Cost per Item from InventoryItem

### DISCUSSION THREAD

**Discussion by kragit** [nmaier95/craft-shopify-product-fetcher#159] [Ideas] [answered]
**Description**

Price and Compare at Price are already imported, but it would also be great to import the "Cost per item" ([unitCost](https://shopify.dev/docs/api/admin-graphql/2023-10/objects/InventoryItem#field-inventoryitem-unitcost)) that's available within the InventoryItem object. That way, some basic profit calculations could be done through Craft.

Screenshot of "Cost per Item" in Shopify:
![costperitem](https://github.com/craftcms/shopify/assets/296050/909d00d9-da7e-4a72-96aa-61fd97203f03)

**Comment by linear** [comment]
<p><a href="https://linear.app/craftcms/issue/PT-1882/enhancement-sync-cost-per-item-from-inventoryitem">PT-1882 Enhancement: Sync Cost per Item from InventoryItem</a></p>

**Comment by nfourtythree** [comment, marked as answer]
Hi @kragit 

Thank you for your feature request. We will add this to our list to take a look at for future improvements.

We will also keep it in mind as we put together the new version of the plugin (#130)

Thanks!

**Reply by kragit** [reply]
Thanks for including it in the newer version, it's working great!
